charts/adminer/values.yaml.gotmpl

```diff
@@ -50,12 +50,13 @@ ingress:
   className: ""
   annotations:
     cert-manager.io/cluster-issuer: "cert-issuer"
+    traefik.ingress.kubernetes.io/router.entrypoints: websecure
   tls:
     - hosts:
-        - k8s.monitoring.{{ requiredEnv "MACHINE_FQDN" }}
+        - {{ requiredEnv "K8S_MONITORING_FQDN" }}
```

Comment:
Q: what is this entrypoint? monitoring.k8s.X?

Reply:
yes